src/kytos/interfaceInfo.vue

```diff
@@ -21,7 +21,50 @@
               <k-property-panel-item :name="key" :value="value" :key="key" v-if="content" v-for="(value, key) in this.metadata"></k-property-panel-item>
           </k-property-panel>
       </k-accordion-item>
-
+      <k-accordion-item title="Available Tags">
+        <div class="metadata_table">
+          <table>
+            <thead>
+              <tr>
+                <th>Type</th>
+                <th>Value Ranges</th>  
+              </tr>
+            </thead>
+            <tbody>
+              <tr v-for="(value, key) in this.available_tags">
+                <td >{{key}}</td>
+                <td >{{value}}</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </k-accordion-item>
+      <k-accordion-item title="Tag range">
```

Comment:
```suggestion
      <k-accordion-item title="Tag ranges">
```

Trivial detail, but if you could pluralize here that'd be great.

Reply:
Fixed